genie-core/src/main/java/com/netflix/genie/core/jobs/workflow/impl/ClusterTask.java

```diff
@@ -95,6 +98,13 @@ public void executeTask(@NotNull final Map<String, Object> context) throws Genie
                 AdminResources.CLUSTER
             );
 
+            // Create the dependencies directory for this id
+            createEntityInstanceDependenciesDirectory(
```

Comment:
At this point you're just creating a directory. If you want some reusable method that's fine but you can just say create a `dependencies` directory within this folder rather than all these arguments